### ISSUE TITLE
FIX:  Montgomery date field not in '%a. %B %d, %Y' format

### ISSUE THREAD

**Issue by NicoleJaneway** [DataKind-DC/capital-nature-ingest#179]
## Current Behavior
When running get_events.py, I got the following warning:  ValueError: time data 'November 22 - December' does not match format '%a. %B %d, %Y'.  But the scraper does run and output events to the .csv.

**Comment by csmcallister** [comment]
@NicoleJaneway thanks for making this issue. I've been meaning to do so for awhile, but it's an edge case.

Here's the full log message for working on it:

```python
Exception schematizing this event date: November 22 - December","<class 'ValueError'> time data 
'November 22 - December' does not match format '%a. %B %d, %Y' File 
""/Users/mcallistercs/Desktop/Github/capital-nature-ingest/events/montgomery.py"" line 150 in 
schematize_event_date    datetime_obj = datetime.strptime(event_date ""%a. %B %d %Y"")   File 
""/Users/mcallistercs/anaconda/lib/python3.6/_strptime.py"" line 565 in _strptime_datetime    tt 
fraction = _strptime(data_string format)   File 
""/Users/mcallistercs/anaconda/lib/python3.6/_strptime.py"" line 362 in _strptime    (data_string 
format))"
```

The date from the website isn't like the others, so the datetime parsers fail. Solution is to handle this case and include the event's url in the log message to make debugging easier.

**Comment by csmcallister** [comment]
closed by d385a9a
